iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/NodeBreakerVoltageLevel.java

```diff
@@ -464,29 +464,13 @@ public boolean isValid(UndirectedGraph<? extends TerminalExt, SwitchImpl> graph,
                 if (terminal != null) {
                     AbstractConnectable connectable = terminal.getConnectable();
                     switch (connectable.getType()) {
-                        case LINE:
-                        case TWO_WINDINGS_TRANSFORMER:
-                        case THREE_WINDINGS_TRANSFORMER:
-                        case HVDC_CONVERTER_STATION:
-                        case DANGLING_LINE:
+                        case LINE, TWO_WINDINGS_TRANSFORMER, THREE_WINDINGS_TRANSFORMER, HVDC_CONVERTER_STATION, DANGLING_LINE -> {
                             branchCount++;
                             feederCount++;
-                            break;
-
-                        case LOAD:
-                        case GENERATOR:
-                        case BATTERY:
-                        case SHUNT_COMPENSATOR:
-                        case STATIC_VAR_COMPENSATOR:
-                            feederCount++;
-                            break;
-
-                        case BUSBAR_SECTION:
-                            busbarSectionCount++;
-                            break;
-
-                        default:
-                            throw new IllegalStateException();
+                        }
+                        case LOAD, GENERATOR, BATTERY, SHUNT_COMPENSATOR, STATIC_VAR_COMPENSATOR, GROUND -> feederCount++;
```

Comment:
adding the ground here means that the following cases are considered as a valid bus:
- a busbar with a ground
- a line and two grounds

I think we don't want that